docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -1826,12 +1826,14 @@
     "\n",
     "1. Use the `recover_configurations()` method to obtain a refined bitstring matrix and probability array based on the average orbital occupancy.\n",
     "1. Use the `postselect_and_subsample()` to collect batches of subsamples to diagonalize over.\n",
-    "1. Then use the batches of subsamples as arguments to the `solve_fermion()` method to obtain an approximation of the ground state."
+    "1. Then use the batches of subsamples as arguments to the `solve_fermion()` method to obtain an approximation of the ground state.\n",
+    "\n",
+    "One important note here is how to address the very first iteration of the configuration recovery loop. Here the average orbital occupancy is not yet available and so only the `postselect_and_subsample()` method is called. This removes any non-physical samples (samples with incorrect Hamming weight) before running the eigenstate solver, `solve_fermion()`. Afterwards, the average orbital occupation is calculated across all batches and used as input to the `recover_configurations()` method which flips individual bits probabilistically based on this average. The probability of flipping an individual bit is based on the distance between the value of the bit and the average orbital occupancy obtained. (See Section **II-A** of the supplementary information in the [SQD paper](https://arxiv.org/abs/2405.05068) for more information.)"
```

Comment:
```suggestion
    "It is important to note how to address the first iteration of the configuration recovery loop. Since the average orbital occupancy is not yet available, only the `postselect_and_subsample()` method is called. This removes any non-physical samples (samples with incorrect Hamming weight) before running the eigenstate solver, `solve_fermion()`. Afterward, the average orbital occupation is calculated across all batches and used as input to the `recover_configurations()` method, which flips individual bits probabilistically based on this average. The probability of flipping an individual bit is based on the distance between the value of the bit and the average orbital occupancy obtained. (See Section **II-A** of the supplementary information in the [SQD paper](https://arxiv.org/abs/2405.05068) for more information.)"
```